runtime/doc/usr_01.txt

```diff
@@ -70,52 +70,18 @@ If the file already exists you probably want to keep it.
 For more info see |vimrc|.
 
 ==============================================================================
-*01.3*	Using the Vim tutor				*tutor* *vimtutor*
+*01.3*	Using the Vim tutor			      	            *tutor*
```

Comment:
It's a completely new implementation with different behavior and whatnot, but at its heart it's still `vimtutor`, so :+1: on the compatibility tag.